static/src/javascripts/projects/common/modules/ui/newAutoupdate.js

```diff
@@ -54,7 +54,7 @@ define([
         };
 
         var scrolledPastTopBlock = function () {
-            return $liveblogBody.offset().top < window.scrollY;
+            return $liveblogBody.offset().top < window.pageYOffset;
         };
         var isLivePage = !(window.location.href.search('[?&]page=') !== -1);
```

Comment:
good point,t hat's what I used and it seems(!) to work
https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/liveblog.js#L156